ledger/daml-on-sql/README.rst

```diff
@@ -0,0 +1,703 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML-on-SQL
```

Comment:
I think the product names are always without dashes.
```suggestion
DAML on SQL
```
Not sure if we should or should not always italicize _DAML on SQL_ to emphasize it as a reference to the product.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6850/commits/f57199d0811464e1f0c13d7b9bd3a98f5873cd7a

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6850/commits/f57199d0811464e1f0c13d7b9bd3a98f5873cd7a